### DISCUSSION TITLE
Componente Search con dropdown

### DISCUSSION THREAD

**Discussion by Payas17** [TiendaNube/nimbus-design-system#54] [Q&A] [answered]
Hola, estábamos buscando un componente que sea un search con dropdown. Tenemos un form con un campo de ciudad que queremos que, al ingresar texto dentro de este campo, busque la ciudad completando el texto. Por ejemplo al buscar "bog", nos muestre en el dropdown "bogota" y cualquier otra ciudad que comience con "bog". En nimbus vimos que existe el componente SearchFilter y Input con type "search" pero creo que no funcionan de la manera que queremos. No usamos el componente Select porque vamos a tener una lista de 1000 ciudades aproximadamente y no queremos que el merchant este buscando entre todas las opciones la ciudad.

¿Hay algún componente que resuelva esto?

Gracias

**Comment by jcampos89** [comment, marked as answer]
Hola @Payas17, como estás?

Por ahora no tenemos ningún componente similar, anteriormente recibimos un pedido similar, pero es un componente complejo y optamos por no priorizarlo, pero si es essencial para ustedes, sienta te libre para armar un PR y lo analisamos.
